code/modules/mob/living/carbon/life.dm

```diff
@@ -129,7 +141,8 @@
 
 	else //Enough oxygen
 		failed_last_breath = 0
-		adjustOxyLoss(-5)
+		if(getOxyLoss())

```

Comment:
this check is here so adjustOxyloss(-5) doesn't call updatehealth() every time we breathe.